assets/app/scripts/controllers/build.js

```diff
@@ -99,8 +99,34 @@ angular.module('openshiftConsole')
           if (!$filter('isIncompleteBuild')(build)){
             delete $scope.buildConfigBuildsInProgress[buildConfigName][buildName];
           }
-        }        
+        }
       }));
+
+      (function initLogs() {
```

Comment:
No, it's fine. Missed that we were calling it in the view, although I think we should remove the reconnect link anyway.